### DISCUSSION TITLE
Dead link

### DISCUSSION THREAD

**Discussion by vulturo26** [DocMartin7644/Chiltern-Route-v2#3] [Q&A]
The link to the Chiltern route on the Open Rails home page leads to 'not found'

**Comment by DocMartin7644** [comment]
I have just checked the link and it is working for me.  Alternatively, you
could try going directly to the route on GitHub at
https://github.com/DocMartin7644/Chiltern-Route-v2

Martin

**Comment by vulturo26** [comment]
Hi thanks for getting back... Sorry, garbled message. The zip link on
Github ( https://github.com/DocMartin7644/Chiltern-Route-v2) leads to 'not
found'...

On Fri, 23 Sept 2022 at 13:38, DocMartin7644 ***@***.***>
wrote:

> I have just checked the link and it is working for me. Alternatively, you
> could try going directly to the route on GitHub at
> https://github.com/DocMartin7644/Chiltern-Route-v2
>
> Martin
>
> On Fri, 23 Sept 2022 at 06:02, vulturo26 ***@***.***> wrote:
>
> > The link to the Chiltern route on the Open Rails home page leads to 'not
> > found'
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <https://github.com/DocMartin7644/Chiltern-Route-v2/discussions/3>, or
> > unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/AXL7AKZOW4J2U6ZYU76PGJLV7WE4XANCNFSM6AAAAAAQT37OQY
> >
> > .
> > You are receiving this because you are subscribed to this thread.Message
> > ID: ***@***.***>
> >
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/DocMartin7644/Chiltern-Route-v2/discussions/3#discussioncomment-3717021>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A3BAWJ7JRY2VL2XQH2FEYVLV7WQDLANCNFSM6AAAAAAQT37OQY>
> .
> You are receiving this because you authored the thread.Message ID:
> <DocMartin7644/Chiltern-Route-v2/repo-discussions/3/comments/3717021@
> github.com>
>


-- 
Stephen Butler
3 Golf Links Court
Tadcaster
LS24 9JD
N. Yorkshire

**Reply by DocMartin7644** [reply]
Not for me, it doesn't.  I can use the link to access my route on GitHub.  It looks as though something strange is happening with your system.

Martin

**Comment by vulturo26** [comment]
More than likely 😀

On Fri, 23 Sept 2022 at 16:29, DocMartin7644 ***@***.***>
wrote:

> Not for me, it doesn't. I can use the link to access my route on GitHub.
> It looks as though something strange is happening with your system.
>
> Martin
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/DocMartin7644/Chiltern-Route-v2/discussions/3#discussioncomment-3718253>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A3BAWJYQEH3QDRMZZCQ34N3V7XEE5ANCNFSM6AAAAAAQT37OQY>
> .
> You are receiving this because you authored the thread.Message ID:
> <DocMartin7644/Chiltern-Route-v2/repo-discussions/3/comments/3718253@
> github.com>
>


-- 
Stephen Butler
3 Golf Links Court
Tadcaster
LS24 9JD
N. Yorkshire

**Reply by DocMartin7644** [reply]
Stephen,

Are you able to go to github.com ?  If so, once there search for my repository docmartin7644/chiltern-route-v2  If you manage to open my repository, click on the green CODE button and you will be given the opportunity to download the route as a zip file.

Much preferred, though, is to do a Google search for GitHub Desktop and install the app.  Using this facility, "clone" the route.  This will allow you to keep up to date with any additions to the route without having to download the whole route again.

Martin

**Comment by vulturo26** [comment]
Thanks - I did eventually download via github. Looking forward to exploring
it. Keep up the good work!
S
